webpack.config.js

```diff
@@ -65,7 +65,23 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					"sass-loader?data=@import 'assets/stylesheets/shared/component';" +
+						'&outputStyle=' + ( bundleEnv === 'production' ? 'compressed' : 'nested' ),
+				]
+			},
+			{
+				test: /\/client\/components\/vertical-nav.+index\.jsx$/,
+				loaders: [
+					'imports?style=./style.scss,withStyles=isomorphic-style-loader/lib/withStyles',
+					'exports?default=withStyles.default( style )( exports.default )'
```

Comment:
Very good point. I think we need to validate the following question:

> Is it possible to create Webpack loader which scans every `*.js` and `*.jsx` files and picks only those that use React. For all those files it should also look if there is a `style.scss` file in the same folder. If all those requirements are met for a single file it should modify this file as above.

Does it make sense? I think it would cover most of the use cases. Unless we have files which use React but don't expose components ...